services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/switches/MetersSpec.groovy

```diff
@@ -358,6 +359,27 @@ class MetersSpec extends BaseSpecification {
         ]
     }
 
+    @Ignore("https://github.com/telstra/open-kilda/issues/2043")
+    def "Try to sync meters for unmetered flow"() {
+        given: "A flow with the 'bandwidth: 0' and 'ignoreBandwidth: true' fields"
+        def availableSwitches = topology.activeSwitches
+        def src = availableSwitches[0]
+        def dst = availableSwitches[1]
+        assumeTrue("Desired switch combination is not available in current topology", src && dst)
+
+        def flow = flowHelper.randomFlow(src, dst)
+        flow.ignoreBandwidth = true
+        flow.maximumBandwidth = 0
+        flowHelper.addFlow(flow)
+
+        when: "resetting meter burst and rate to default"
+        northbound.resetMeters(flow.id)
+//        TODO add check response: body or message
+
+        then: "Delete the created flow"
```

Comment:
In 'then' block, we should verify that meters are actually reset. Deleting flow should be a kind of cleanup and should be moved to 'and' block.

Reply:
Done.